lib/app_web/components/layouts/app.html.heex

```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Phoenix Todo List</title>
     <link rel="stylesheet" href={~p"/assets/app.css"} />
+    <script src="https://unpkg.com/browse/@hotwired/turbo@7.2.4/dist/turbo.es2017-esm.js">
```

Comment:
why does this need to be added in _twooo_ places? 🤔

Reply:
Added on both when it wasn't working and forgot to take it out. Fixed.

Reply:
thanks for updating. 👌